### PR TITLE
Moving service worker job related code to its own file.

### PR DESCRIPTION
**Pull request by ghost**
b/286597506

Change-Id: I0ca3ac56d3f1e1acdc6fd2cd8faac34a463b8d61

### CODE REVIEW

**Review by sherryzy — COMMENTED** [review]
Nice first step! This looks good to me. Thanks for doing the refactor job. Would love @jellefoks to take a look as well.

**Review by jellefoks — APPROVED** [review]
Looks great. Thanks for taking the first stab at this. I hope that maybe sometime later we won't need a 'friend class' between these two either.

File: internal [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/local/google/home/aee/cobalt/cobalt_src/internal
```
Comment (kaidokert) [inline internal:1]: I didn't review the rest but it looks like there's something wrong here with "internal" sneaking in.

**Review by kaidokert — CHANGES_REQUESTED** [review]
Please make sure the /internal directory/submodule doesn't get in, git submodules cause a lot of havoc.

Other than that looks like a great refactor

**Review by kaidokert — APPROVED** [review]

**Pull request merged**